File: js/SumOfDigit.js

function SumOfDigit(num) {
    let sum = 0;

    let temp = num;

    while (temp > 0) {
        let remainder = temp % 10;

        sum = sum * 10 + remainder;

        temp = Math.floor(temp / 10);

    }

    if (num == sum) {
       
        console.log("palin");
        
        
    } else {
        console.log('not palin');
        
    }

    return sum;
}

let res = SumOfDigit(152)

console.log(res);

File: SumOfDigit.cpp
#include <iostream>

using namespace std;

int main(){
    int num, sum = 0, rem, temp;

    cin>> num;

    temp = num;

    while (temp != 0)
    {
        rem = temp % 10;
        sum = sum + rem;
        temp = temp / 10;
    }
    
    cout<< sum;
}